routes/index.js

```diff
@@ -346,6 +346,35 @@ module.exports = function (passport) {
 
   });
 
+
+  router.post('/userManagement/demoteAdmin/:badge', checkAuth, function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    var badgeID = req.params.badge;
+
+    dbAPI.validateUser(badgeID).then(function (ret) {
+      console.log(ret.dataValues);
+      console.log("USER STATUS " + ret.status);
+
+      if (ret.status == "Admin") {
+        dbAPI.modifyUser(badgeID, { status: "Manager", password: null }).then(function (result) {
+          if (result == undefined) {
+            console.log("Error");
+          }
+          else {
+            console.log("demotion changed successfully");
```

Comment:
Please add DB logging here instead of `console.log`, one event class is going to be 'administration' and the other is going to be `privilege`. The entity field for `administration` with contain the admin badge ID responsible to demoting the other admin. You can access this through `req.user.badge`. For the `privilege` event class the entity field will just be `badgeID`. Please refer to my DB logging pull request if you are confused, or message me and I can explain more.